### PR TITLE
Use core 2.19.1

### PR DESCRIPTION
**Pull request by johnkerl**
**Issue and/or context:**

**Changes:**

**Notes for Reviewer:**

### CODE REVIEW

File: apis/r/tools/get_tarball.R [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+tiledb_core_sha1 <- "29ceb3e7"
 
 if ( ! dir.exists("inst/") ) {
     stop("No 'inst/' directory. Exiting.", call. = FALSE)
 }
 
 makeUrl <- function(arch, ver=tiledb_core_version, sha1=tiledb_core_sha1) {
-    sprintf("https://github.com/TileDB-Inc/TileDB/releases/download/%s/tiledb-%s-%s-%s.tar.gz", ver, arch, ver, sha1)
+-    sprintf("https://github.com/TileDB-Inc/TileDB/releases/download/%s/tiledb-%s-%s-%s.tar.gz", ver, arch, ver, sha1)
+    sprintf("https://github.com/TileDB-Inc/TileDB/releases/download/2.19.1/tiledb-2.19.1-2.19.1-2.19.1.tar.gz", ver, arch, ver, sha1)
```
Comment (eddelbuettel) [inline apis/r/tools/get_tarball.R:14]: You may have left one line.

**Review by eddelbuettel — COMMENTED** [review]
See inline.

File: apis/r/tools/get_tarball.R (hunk 1 above)
Comment (johnkerl) [reply apis/r/tools/get_tarball.R]: 🙏

Comment (johnkerl) [reply apis/r/tools/get_tarball.R]: https://github.com/single-cell-data/TileDB-SOMA/pull/2084/commits/ee2c8ccb12a08db218aaa1e5b0a9a9feae98ae15

Comment (johnkerl) [reply apis/r/tools/get_tarball.R]: @eddelbuettel resolved

Comment (johnkerl) [reply apis/r/tools/get_tarball.R]: `scripts/update-tiledb-version.py i.j.k` _always_ requires a manual edit on that file -- which I had left half-done, and which you spotted (thank you)

**Review by eddelbuettel — APPROVED** [review]
Looks good and seems to behave in CI.  Fingers crossed...

**Pull request merged**